### PR TITLE
Suce 501 fallback language

### PR DESCRIPTION
**Pull request by gradam**

### CODE REVIEW

**Comment by TrebuhD** [comment]
Does this work? The generated files should be empty, that is like that:
```
{
    "pl": {}
}
```

File: client/localization/extract_static_strings.py [hunk 1]
```diff
@@ -8,6 +8,9 @@
 from bs4 import BeautifulSoup
 from bs4.element import NavigableString
 
+AVAILABLE_LANGUAGES = ['en', 'pl']
```
Comment (TrebuhD) [inline client/localization/extract_static_strings.py]: should we not be getting the list of languages from the JSON file we're loading into arangodb?

File: client/localization/extract_static_strings.py [hunk 2]
```diff
@@ -8,6 +8,9 @@
...
+AVAILABLE_LANGUAGES.remove('en')
```
Comment (TrebuhD) [inline client/localization/extract_static_strings.py]: these two lines don't make sense.

File: client/localization/extract_static_strings.py (hunk 1 above)
Comment (gradam) [reply client/localization/extract_static_strings.py]: We can't do that easily from the client

File: client/localization/extract_static_strings.py (hunk 2 above)
Comment (gradam) [reply client/localization/extract_static_strings.py]: not true. This way you don't have to care about which language is 'source' one so you just write list of all available languages

**Pull request merged**